cmplxs/cmplxs.go

```diff
@@ -85,6 +85,22 @@ func Count(f func(complex128) bool, s []complex128) int {
 	return n
 }
 
+// Complex fills each of the elements of dst with the complex number
+// constructed from the corresponding elements of real and imag.
+// A panic will occur if the lengths of arguments do not match.
```

Comment:
it seems to me we usually don't use much passive form for that kind of documentation.
_i.e.:_ I think we usually word it as:
```
// Complex panics if ...
```

Reply:
This wording is lifted directly from another function.